src/statusTile/updateStatusTileScope.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+const { getCurrentScope } = require('../editorInterface');
+const { getAllScopes } = require('../atomInterface');
+
+const updateStatusTileScope = (element: HTMLElement, editor: TextEditor) => {
+  if (element) {
```

Comment:
prefer guard clauses for exceptional scenarios, this way you don't have to wrap the entire implementation inside of an if statement.

```js
if (!element) return;

// ...
``